test/Controller/ArticleControllerTest.php

```diff
@@ -4707,6 +4707,26 @@ public function it_displays_recommendations()
                         ],
                         'authorLine' => 'Author One et al.',
                     ],
+                    [
```

Comment:
Can we add a test to check that reviewed preprints if returned from the recommendations api can occupied the related item and further reading?